client/landing/stepper/declarative-flow/trial-ecommerce-flow.ts

```diff
@@ -0,0 +1,62 @@
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useSiteSetupFlowProgress } from '../hooks/use-site-setup-flow-progress';
+import { useSiteSlugParam } from '../hooks/use-site-slug-param';
+import { ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import ProcessingStep, { ProcessingResult } from './internals/steps-repository/processing-step';
+import SiteCreationStep from './internals/steps-repository/site-creation-step';
+import { Flow, ProvidedDependencies } from './internals/types';
+
+const ecommerceTrialFlow: Flow = {
+	name: 'ecommerce-trial',
+
+	useSteps() {
+		return [
+			{ slug: 'siteCreationStep', component: SiteCreationStep },
+			{ slug: 'processing', component: ProcessingStep },
+		];
+	},
+	useStepNavigation( currentStep, navigate ) {
```

Comment:
I think we also need a safeguard to make sure users are actually logged in already. For reference, we do that [here](https://github.com/Automattic/wp-calypso/blob/4af60e6e499d81ac9e9a49e29f817ad594311fb5/client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts#L63-L94) in the tailored eCommerce flow. We can definitely fold that in later, but I want to make sure we handle that case.